constants.js

```diff
@@ -43,6 +43,7 @@ export const aiHost = `https://ai.webaverse.com`;
 export const web3MainnetSidechainEndpoint = 'https://mainnetsidechain.exokit.org';
 export const web3TestnetSidechainEndpoint = 'https://testnetsidechain.exokit.org';
 export const worldUrl = 'worlds.webaverse.com';
+export const discordAuthUrl = `https://discord.com/api/oauth2/authorize?client_id=684141574808272937&redirect_uri=${window.location.origin}%2Flogin&response_type=code&scope=identify`
```

Comment:
Hm, I don't think this should be a constant, it is very specific to Discord logic.

`discordClientId: 684141574808272937` make sense to me here, but the URL synthesis probably belongs with the login code.